config/examples/Creality/Ender-3/BigTreeTech SKR E3-DIP V1.1/Configuration_adv.h

```diff
@@ -1164,7 +1164,7 @@
 
   // The standard SD detect circuit reads LOW when media is inserted and HIGH when empty.
   // Enable this option and set to HIGH if your SD cards are incorrectly detected.
-  //#define SD_DETECT_STATE HIGH
+  #define SD_DETECT_STATE HIGH
```

Comment:
Is this really correct? Because on my Ender 3 with BigTreeTech SKR E3-DIP V1.0 board I need to leave this line comment out. With `SD_DETECT_STATE HIGH` set, the SD card does not work and the printer shows "SD Init Fail" when I remove the card and "Media Removed" when I insert it.

As far as I can tell v1.1 of the board has no changes in this regard:
> What's new with Board version V1.1:
>
> - Modified the SERVO port (hardware of board -- check PDF)
> - Firmware is compatible for both v1.0 and v1.1

https://github.com/bigtreetech/BIGTREETECH-SKR-E3-DIP-V1.0

See also the original config from BIGTREETECH's repository: https://github.com/bigtreetech/BIGTREETECH-SKR-E3-DIP-V1.0/blob/c93d0309a42848e0fa27f005e9b2031736be7e61/Firmware/Marlin-2.0.7.2-SKR-E3-DIP/Marlin/Configuration_adv.h#L1171